src/main/java/PlayerData.java

```diff
@@ -0,0 +1,150 @@
+/** relevant imports*/
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+/**
+ * The PlayerData class
+ */
+public class PlayerData implements Serializable {
+    /** Stores the health of the PlayerData object */
+    private int health;
+
+    /** Stores the attack power of the PlayerData object */
+    private int attackPower;
+
+    /** The current event */
+    private int eventID;
+
+    /** The username of the player */
+    private String username;
+
+    /** The inventory of the PlayerData stores the name of the item mapped to the ItemData object */
+    private HashMap<String, ArrayList<ItemData>> inventory;
```

Comment:
Inventory can be Map so that if we change inventory to another type of map later on, we don't have to change the code.

Reply:
We can still make inventory a HashMap, but we can just set the type to be Map for now?

Reply:
Maybe we can talk about this more at the next meeting.

Reply:
Yes, we can discuss it on Monday